src/main.cpp

```diff
@@ -118,8 +118,10 @@ int main(const int argc, const char *argv[])
         inset_state.apply_albers_projection();
       }
     } else if (output_equal_area) {
-      std::cerr << "ERROR: Input GeoJSON is not a longitude-latitude map."
-                << std::endl;
+      std::cerr
+        << "ERROR: Input GeoJSON is not a longitude-latitude map. Therefore, "
+           "it is not possible to produce an equal-area map."
```

Comment:
I feel we should not remove it; this way we know that the `--output_equal_area` flag did not work for the given map. It is true that the statement contains redundant information because the first statement technically implies the second statement given the context. However, for a new user, it might not be obvious; and we do plan to display error messages to the users on the website.